### PR TITLE
fixes #22799 - setting to indicate disconnected mode

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

**Comment by jlsherrill** [comment]
We may want to discuss how the new UI will even use this setting, since only the admin user can fetch settings IIRC.

**Comment by stbenjam** [comment]
The card mentioned to include it in the subscriptions list API, would that be an option? The dynamic javascript thingy also sounds fine

**Comment by waldenraines** [comment]
> Bastion did something where it would embed certain settings in a javascript file (rendered on the server), maybe the react stuff can do that. @waldenraines ping!

Yeah, for bastion we did this:  https://github.com/Katello/bastion/blob/master/app/views/bastion/layouts/assets.html.erb#L25

And we could do something similar for react or we could have an API to get settings or include it in the subscriptions API for now and then move it later.  Any of these approaches is fine with me.  I would prefer to get it from some sort of API though rather than the way we injected it in bastion.

**Comment by stbenjam** [comment]
@akofink is going to be writing the API, what do you think? Could you incorporate this setting's value?

**Comment by stbenjam** [comment]
[test katello]

**Comment by akofink** [comment]
I'll update https://github.com/Katello/katello/pull/7197 to check this value.

**Comment by stbenjam** [comment]
That sounds ok to me.  I'm fine if you want to take my one-line of a PR and incorporate it into yours, if it's easier.

I'm also not sure if this is named well :man_shrugging:

**Comment by akofink** [comment]
Nah, let's merge this. :)

File: app/models/setting/content.rb [hunk 1]
```diff
@@ -37,6 +37,8 @@ def self.load_defaults
                  20, N_('Accept action timeout')),
         self.set('content_action_finish_timeout', N_("Time in seconds to wait for a Host to finish a remote action"),
                  3600, N_('Finish action timeout')),
+        self.set('content_disconnected', N_("A sever operating in disconnected mode does not communicate with the Red Hat CDN."),
```
Comment (bbuckingham) [inline app/models/setting/content.rb]: s/sever/server/

Comment (stbenjam) [reply app/models/setting/content.rb]: Thanks :)

**Comment by waldenraines** [comment]
[test katello]

**Comment by waldenraines** [comment]
[test katello]

**Comment by waldenraines** [comment]
[test katello]

**Pull request merged**